guides/source/active_record_callbacks.md

```diff
@@ -656,66 +1008,132 @@ class User < ApplicationRecord
 
   private
     def log_user_saved_to_db
-      puts 'User was saved to database'
+      Rails.logger.info("User was saved to database")
     end
 end
 ```
 
 ```irb
-irb> @user = User.create # prints nothing
+irb> user = User.create # prints nothing
 
-irb> @user.save # updating @user
+irb> user.save # updating @user
 User was saved to database
 ```
 
 ### `after_save_commit`
 
-There is also [`after_save_commit`][], which is an alias for using the `after_commit` callback for both create and update together:
+There is also [`after_save_commit`][], which is an alias for using the
+`after_commit` callback for both create and update together:
 
 ```ruby
 class User < ApplicationRecord
   after_save_commit :log_user_saved_to_db
 
   private
     def log_user_saved_to_db
-      puts 'User was saved to database'
+      Rails.logger.info("User was saved to database")
     end
 end
 ```
 
 ```irb
-irb> @user = User.create # creating a User
+irb> user = User.create # creating a User
 User was saved to database
 
-irb> @user.save # updating @user
+irb> user.save # updating user
 User was saved to database
 ```
 
 ### Transactional Callback Ordering
 
-By default, callbacks will run in the order they are defined. However, when
-defining multiple transactional `after_` callbacks (`after_commit`,
-`after_rollback`, etc), the order could be reversed from when they are defined.
+By default (from Rails 7.1), transaction callbacks will run in the order they are defined.
+
+However, in prior versions of Rails, when defining multiple transactional
+`after_` callbacks (`after_commit`, `after_rollback`, etc), the order in which
+the callbacks were run was reversed.
 
 ```ruby
 class User < ActiveRecord::Base
-  after_commit { puts("this actually gets called second") }
-  after_commit { puts("this actually gets called first") }
+  after_commit { Rails.logger.info("this actually gets called second") }
+  after_commit { Rails.logger.info("this actually gets called first") }
+end
+```
+
+We can change that reverse behaviour by setting the [following
```

Comment:
Will this be necessary for the guides? Either way, calling this change should be the opposite. We would have to update the example below in that case as well.

Reply:
I think it's a good point to note, but I've updated the example, thanks for pointing it out!